Include/internal/pycore_pyatomic_ft_wrappers.h

```diff
@@ -63,6 +64,7 @@ extern "C" {
     _Py_atomic_store_uint32_relaxed(&value, new_value)
 
 #else
+#define FT_ATOMIC_ADD_UINT64(value, new_value) value += new_value
```

Comment:
These implementations don't match exactly.

* `_Py_atomic_add_uint64` returns the *previous* value
* `value += new_value` evaluates to the *new* value 

There's just one use of this macro, and I think it's awkward to make their semantics the same, so consider just using a `#ifdef Py_GIL_DISABLED` below in `enter_task`